app/views/wpcc/your_details/new.html.erb

```diff
@@ -42,10 +39,7 @@
         <div class="details__field">
           <div class="details__field-label">
             <%= f.label :gender, t('wpcc.details.gender.label'), class: 'form__label-heading' %>
-            <button type="button" class="popup-tip__button" data-dough-popup-trigger>
-              <span aria-hidden="true">i</span>
-              <span class="visually-hidden"><%= t('wpcc.details.tooltip_show') %></span>
-            </button>
+            <%= render 'wpcc/shared/popup_tip_trigger' %>
```

Comment:
@tomas-stefano OK I understand your reasoning. The shared folder was already created for a couple of other things so I just added this one to that area. If I understand correctly the simplest interpretation of what you are suggesting is to create a "tooltips" folder and rename the partials to, say, "show" and "close" within that. Would that be right?